src/Framework/AssemblyLoadBuildEventArgs.cs

```diff
@@ -77,7 +78,11 @@ public override string Message
                 if (RawMessage == null)
                 {
                     string? loadingInitiator = LoadingInitiator == null ? null : $" ({LoadingInitiator})";
+#if FEATURE_ASSEMBLYLOADCONTEXT
+                    RawMessage = FormatResourceStringIgnoreCodeAndKeyword("TaskAssemblyLoadedWithAssemblyLoadContext", LoadingContext.ToString(), loadingInitiator, AssemblyName, AssemblyPath, MVID.ToString(), AppDomainDescriptor ?? DefaultAppDomainDescriptor);
+#else
                     RawMessage = FormatResourceStringIgnoreCodeAndKeyword("TaskAssemblyLoaded", LoadingContext.ToString(), loadingInitiator, AssemblyName, AssemblyPath, MVID.ToString(), AppDomainDescriptor ?? DefaultAppDomainDescriptor);
+#endif
```

Comment:
Technically this can be executed in different execution context (e.g. binlog viewer) - so ideally the LoadContext vs AppDomain info would be already part of the EventArgs. That would however require a version contract update - and it's not worth it. Just mentioning it here :-)

Reply:
Just to be clear, there's nothing you want me to do regarding this comment, right?

Reply:
Sorry - it was rather a confusing self-mumbling :-)
All good here